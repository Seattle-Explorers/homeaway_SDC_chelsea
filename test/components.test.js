// const React = require('react');
// const { mount } = require('enzyme');
// const generateListing = require('../util/generateListing');
// const Description = require('../client/src/components/Description');

describe('component tests', () => {
  test('it should render a description component', () => {
    // const imagesPath = path.join(__dirname, 'testImages');
    // const listing = generateListing('001', '001', imagesPath);

    expect(true).toBe(true);
  });
});
