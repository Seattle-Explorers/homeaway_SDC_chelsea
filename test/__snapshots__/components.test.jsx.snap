// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component tests it should render a description component 1`] = `
<div>
  <body
    className=""
  />
  <div>
    <div>
      Cozy apartment in Portland
    </div>
    <div>
      1
    </div>
    <div>
      1
    </div>
    <div>
      1
    </div>
  </div>
  <div>
    <img
      alt="Devyn Macejkovic"
      src="https://github.com/Team-Danger/FEC-Description-Component/util/profileImages/001.jpg"
    />
    <div>
      Devyn Macejkovic
    </div>
  </div>
  <div>
    Minima perspiciatis praesentium aperiam. Ut occaecati deserunt voluptatem suscipit. Enim minus nemo fugit et iure explicabo et commodi consequuntur.
 
Nesciunt dignissimos eos temporibus temporibus autem consequatur ea laudantium esse. Qui occaecati omnis quis amet voluptatum. Et libero ea et odio. Qui velit qui velit. Provident non fugiat dolorum amet a a aut nihil vel.
 
Excepturi eligendi perferendis quo minus et asperiores nam. Blanditiis consequuntur esse autem. Eligendi aut alias cum consequatur pariatur inventore omnis. Sapiente aliquid illo cumque cupiditate quos cum aut placeat. Ut est ut error quo repellat id aliquam eos.
 
Sunt est itaque ad aut. Quos quis laboriosam molestias quisquam ratione aut est omnis necessitatibus. Ut consectetur voluptas nam praesentium voluptates libero neque.
 
Temporibus quia officiis cumque eos ut nemo occaecati voluptatibus dolor. Totam praesentium ducimus tempore qui rerum autem atque. Esse est inventore autem eligendi aliquid. Voluptates quis quo. Eveniet deleniti quam sed totam nam et.
  </div>
  <div>
    <div>
      Common Space
    </div>
    <div>
      <div>
        <div>
          1
        </div>
        <div>
          Double
        </div>
      </div>
    </div>
  </div>
  <div>
    <div>
      icon here
    </div>
    <div>
      Essentials
    </div>
    <div>
      
    </div>
  </div>
  <div>
    <div>
      icon here
    </div>
    <div>
      Wifi
    </div>
    <div>
      
    </div>
  </div>
  <div>
    <div>
      icon here
    </div>
    <div>
      TV
    </div>
    <div>
      
    </div>
  </div>
  <div>
    <div>
      icon here
    </div>
    <div>
      Heat
    </div>
    <div>
      
    </div>
  </div>
  <div>
    <div>
      icon here
    </div>
    <div>
      Air Conditioning
    </div>
    <div>
      
    </div>
  </div>
</div>
`;
